src/controllers/ContentController.ts

```diff
@@ -1,19 +1,17 @@
 /* eslint-disable no-prototype-builtins */
```

Comment:
This file changes only related to renaming 'Layout' to 'LayoutSystem' also in some places it needs to use a new `Layout` class, as we need to create new layouts under the hood in case if config says so.